src/Camera.lua

```diff
@@ -0,0 +1,145 @@
+local Timer = require("src/Timer")
+
+local Camera = {}
+
+-- note that these onClick functions are used by wrapper functions in Global.lua
+function Camera.onCourtClick(player, value, id)
```

Comment:
Remove the `Camera.` and these functions will be global. This keeps the code more organized, otherwise the Global.lua file will be more bloated than it already is.